### PR TITLE
Add new domain for handling parameter changes 

### PR DESCRIPTION
**Pull request by ManfredKarrer**

### CODE REVIEW

File: src/main/proto/pb.proto [hunk 1]
```diff
@@ -1412,6 +1417,13 @@ message MyVote {
     string reveal_tx_id = 5;
 }
 
+message ParamChangeEvent {
+    // We use odingal instead of Enum as enaums in PB are a pain... as well we want to be more flexible at changes
```
Comment (sqrrm) [inline src/main/proto/pb.proto:1421]: The spelling in this comment could use some love :)

**Review by sqrrm — APPROVED** [review]

File: src/main/proto/pb.proto (hunk 1 above)
Comment (ManfredKarrer) [reply src/main/proto/pb.proto:1421]: ups... ;-) was late...

Comment (ManfredKarrer) [reply src/main/proto/pb.proto:1421]: I assume rest is utACK...

Comment (sqrrm) [reply src/main/proto/pb.proto:1421]: Yes, utACK, forgot to add that.

**Pull request merged**